server.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const app = express();
```

Comment:
I named this `server` in the past lab. I'd keep it as that to be consistent and not confuse students.